pkg/operator/ceph/controller/predicate.go

```diff
@@ -364,24 +364,6 @@ func WatchControllerPredicate() predicate.Funcs {
 				if isUpgrade {
 					return true
 				}
-
-			case *bktv1alpha1.ObjectBucketClaim:
```

Comment:
Why moving this code out to `WatchPredicateForNonCRDObject`?

Reply:
was requested by @BlaineEXE

Reply:
@BlaineEXE do you remember why?

Reply:
This is a CRD so its right place is in `WatchControllerPredicate()` IMO. Especially since it's called via `&source.Kind{Type: &bktv1alpha1.ObjectBucketClaim{}}` in the watcher.